port_ocean/core/handlers/port_app_config/models.py

```diff
@@ -12,6 +12,7 @@ class EntityMapping(BaseModel):
     team: str | None
     properties: dict[str, str] = Field(default_factory=dict)
     relations: dict[str, str] = Field(default_factory=dict)
+    items_to_parse: str | None = Field(alias="itemsToParse")
```

Comment:
pop it one out to the PortResourceConfig

Reply:
you sure? in the webhook service from what i saw its in this level and also [here](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/)

Reply:
Yes i try to follow the custom webhook struct